arkouda/pdarraysetops.py

```diff
@@ -142,11 +142,14 @@ def in1d(pda1 : Union[pdarray,Strings,'Categorical'], pda2 : Union[pdarray,Strin
     array([False, True])
     """
     from arkouda.categorical import Categorical as Categorical_
-    if isinstance(pda2, pdarray) or isinstance(pda2, Strings) or isinstance(pda2, Categorical_):
-        from arkouda.dtypes import int_scalars, structDtypeCodes
-        if pda2.size == 0:  # If pda2 is empty, in1d shouldn't do anything.
-            repMsg = generic_msg(cmd="create", args="int64 0")
+    if isinstance(pda1, pdarray) or isinstance(pda1, Strings) or isinstance (pda1, Categorical_):
+        # While isinstance(thing, type) can be called on a tuple of types, this causes an issue with mypy for unknown reasons.
+        if pda1.size == 0:
+            repMsg = generic_msg(cmd="create", args="bool 0")
```

Comment:
This could be simplified by using the same construction as below: `return zeros(0, dtype=bool)`

Reply:
That had occurred to me but I didn't change since I wasn't sure if `zeros` would be happy with 0 length; changing now.